vm/src/vm.rs

```diff
@@ -941,21 +941,45 @@ impl VirtualMachine {
         }
     }
 
+    pub fn _isinstance(&self, obj: &PyObjectRef, cls: &PyObjectRef) -> PyResult<bool> {
+        if let Ok(typ) = PyTypeRef::try_from_object(self, cls.clone()) {
+            Ok(obj.isinstance(&typ))
+        } else {
+            Err(self.new_type_error(format!(
+                "isinstance() arg 2 must be a type or tuple of types, not {}",
+                cls.class()
+            )))
+        }
+    }
+
     /// Determines if `obj` is an instance of `cls`, either directly, indirectly or virtually via
     /// the __instancecheck__ magic method.
-    pub fn isinstance(&self, obj: &PyObjectRef, cls: &PyTypeRef) -> PyResult<bool> {
+    pub fn isinstance(&self, obj: &PyObjectRef, cls: &PyObjectRef) -> PyResult<bool> {
         // cpython first does an exact check on the type, although documentation doesn't state that
-        // https://github.com/python/cpython/blob/a24107b04c1277e3c1105f98aff5bfa3a98b33a0/Objects/abstract.c#L2408
+        // https://github.com/python/cpytho(n/blob/a24107b04c1277e3c1105f98aff5bfa3a98b33a0/Objects/abstract.c#L2408
```

Comment:
typo?

Reply:
Thanks! I applied it 🥳

Reply:
Will you check it again? It seems something didnt' work.

Reply:
```suggestion
        // https://github.com/python/cpython/blob/a24107b04c1277e3c1105f98aff5bfa3a98b33a0/Objects/abstract.c#L2408
```

Reply:
omg .. I had been thought I modified it. Sorry. I upload again.